src/multiplayer-servers/multiplayer-services/server-protection/protocols.md

```diff
@@ -0,0 +1,68 @@
+# Ports and Protocols
+
+Game server ports are often vulnerable to attacks. 
+GameFabric offers Protocols to protect your game server traffic with powerful SteelShield mitigations.
+
+## Pre-requisites
+
+In order to protect the your game server traffic, your GameFabric installation requires:
+
+- Protection support enabled, usually indicated by the presence of the Protection/SteelShield navigation item.
+- Sites that support SteelShield, e.g. bare-metal.
```

Comment:
I didn't want to be too specific, but give an idea why some sites support it, while others don't. It might also change and gets outdated pretty quick, at least I assume so. And I also don't want to give attackers an indicator which game servers would be more vulnerable as this documentation is also public. 

WDYT?

Reply:
Then I would suggest just removing the `e.g. bare-metal`